components/Radio/src/stories/bem.stories.mdx

```diff
@@ -33,9 +33,9 @@ import readme from "../../README.md";
       <label class="denhaag-form-control-label">
```

Comment:
{() => (
Volgens mij mag de react uit de bem, hiervoor is een aparte react.stories.mdx file, inlcusief de comment {/* eslint-disable jsx-a11y/label-has-associated-control */}
jsx = react namelijk